nnpdf_data/nnpdf_data/new_commondata/STAR_2013_1JET_510GEV/metadata.yaml

```diff
@@ -0,0 +1,50 @@
+setname: "STAR_2013_1JET_510GEV"
+
+nnpdf_metadata:
+  experiment: "STAR"
+  nnpdf31_process: "JETS"
+
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1949588"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1949588"
+  version: 1
+
```

Comment:
Please add arXiv reference.

Reply:
done in 1e39f34118574822597f9646a350c2aca23daf05